src/ring_theory/finiteness.lean

```diff
@@ -284,7 +300,7 @@ lemma fg_restrict_scalars {R S M : Type*} [comm_semiring R] [semiring S] [algebr
 begin
   obtain ⟨X, rfl⟩ := hfin,
   use X,
-  exact (submodule.restrict_scalars_span R S h ↑X).symm
+  exact submodule.span_eq_restrict_scalars R S M X h
```

Comment:
I'm pretty sure this lemma doesn't exist any more.

Reply:
Sorry, shouldn't have copied the whole file (I used "edit file" on GitHub).